reports.js

```diff
@@ -57,6 +58,18 @@ async function getContent(url, type) {
   });
 }
 
+/**
+ * @return {string} last month date as yyyymm
+ */
+function lastReportDate() {
+  const reportDate = new Date();
+  // needs a bit more than a month https://github.com/jenkins-infra/infra-statistics/blob/master/Jenkinsfile#L18
+  reportDate.setDate(reportDate.getDate() - 35);
+  const month = reportDate.getMonth() + 1; // January: 0 -> 1
```

Comment:
I suspect this is fixed width, so you probably want to pad it with 0
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart

Reply:
oh you did it with prefix, i see below